openapi.yaml

```diff
@@ -551,15 +551,9 @@ paths:
       - Account
       summary: Credit Card Add/Edit
       description: |
-        Adds/edit credit card information to your Account.
+        **DEPRECATED**. Please use Payment Method Add ([POST /account/payment-methods](/docs/api/account/#payment-method-add)).
 
-        Only one credit card can be associated with your Account, so using this
-        endpoint will overwrite your currently active card information with the
-        new credit card.
-
-        To use this endpoint, you must have a valid `zip` entered for your Account.
-        Use the Account Update ([PUT /account](/docs/api/account/#account-update))
-        endpoint to enter a new zip code.
+        Adds a credit card Payment Method to your account and sets it as the default method.
```

Comment:
In this comment, it is stated "and sets it as the default method"

However, in another edit below, it states "with the option to set it as the default method."

This should be clarified to the correct statement in whichever place is incorrect.

Reply:
This describes the behavior of the deprecated `PUT /account/credit-card` endpoint, which takes in a different request body that doesn't include an `is_default` field, so it currently sets the payment method as the default automatically in the backend.